zeitschrift-fur-kunstgeschichte.csl

```diff
@@ -204,12 +237,25 @@
           </group>
         </else>
       </choose>
+      <choose>
+        <if variable="URL" match="any">
+          <group>
+            <text variable="URL" prefix=", "/>
+            <text term="accessed" prefix=" ("/>
+            <date variable="accessed" prefix=" " delimiter="." suffix=")">
+              <date-part name="day"/>
+              <date-part name="month" form="numeric"/>
+              <date-part name="year"/>
+            </date>
+          </group>
+        </if>
+      </choose>
     </layout>
   </citation>
   <bibliography et-al-min="3" et-al-use-first="1" subsequent-author-substitute="&#8212;&#8212;&#8212;" entry-spacing="0" hanging-indent="true">
     <sort>
-      <key macro="author"/>
-      <key variable="issued"/>
+      <key macro="contributors-long" names-min="3" names-use-first="3"/>
```

Comment:
This is the code that I copied from "Geistes- und Kulturwissenschaften (Heilman)", I did not change the rule intentionally. Is it such a big mistake?
Well I guess names-min="3" and names-use-first="3" are unnecessary.